controllers/datadogagent/feature/liveprocess/feature.go

```diff
@@ -0,0 +1,118 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
It'd be good to add an integration test for this feature in https://github.com/DataDog/datadog-operator/blob/main/controllers/datadogagent_controller_v2_test.go